src/utils/__tests__/azure_storage.test.ts

```diff
@@ -8,9 +8,12 @@ import { isNone } from "fp-ts/lib/Option";
 import {
   BlobNotFoundCode,
   getBlobAsObject,
-  getBlobAsText
+  getBlobAsText,
+  getBlobAsTextWithError
```

Comment:
Why importing it, since it's not used? Is there a non-committed test?

Reply:
no, but I'm adding some test. thank you!